indico/modules/receipts/controllers/event.py

```diff
@@ -104,6 +166,10 @@ def _check_access(self):
 
     def _get_custom_fields(self):
         custom_fields = {f['name']: self.custom_fields_raw.get(f['name']) for f in self.template.custom_fields}
+        if any(not custom_fields[f['name']].startswith('event://')
+               for f in self.template.custom_fields
+               if f['type'] == 'image' and custom_fields[f['name']]):
+            abort(422)
```

Comment:
should we log attempts to use real URLs here? (similar to what you're doing in the `sandboxed_url_fetcher`)

Reply:
i think this validation is important if the instance has external URLs enabled because of some templates, but they're not intended to be used on this particular template (for instance if it is intended to be available to a wider audience). but yes it's probably not a big concern

Reply:
Good point. Makes sense to ensure it's only `event://` there.